tsdb/functions_test.go

```diff
@@ -529,17 +529,17 @@ func TestMapTop(t *testing.T) {
 				values: []testPoint{
 					{"", 20, int64(99), map[string]string{"host": "a"}},
 					{"", 10, int64(53), map[string]string{"host": "a"}},
-					{"", 10, int64(99), map[string]string{"host": "a"}},
+					{"", 10, int64(99), map[string]string{"host": "b"}},
 				},
 			},
 			exp: positionOut{
 				callArgs: []string{"host"},
 				points: PositionPoints{
-					PositionPoint{10, int64(99), map[string]string{"host": "a"}},
+					PositionPoint{10, int64(99), map[string]string{"host": "b"}},
```

Comment:
Changed the tag because it was a tag top query before and the new behavior made the test fail. Changed the query, forgot to change this back.